### DISCUSSION TITLE
Triggering completion after reboot does not auto-prompt to unlock databases

### DISCUSSION THREAD

**Discussion by stdedos** [keepassxreboot/keepassxc#9924] [Q&A] [answered]
I would expect that, if KeePassXC is open, and the Browser-Addon triggers a _completion_, then KeePassXC would auto-target the database-in-question, and trigger an unlock.

It's not such a pain, but I'd like the DWIM.
I am mostly asking if somehow I have done something wrong with the configuration.

KeePassXC is set to:
![image](https://github.com/keepassxreboot/keepassxc/assets/133706/9c383005-967a-4400-bb26-63a85c896d09)

This is part of my config file:
```
   1   │ [General]
   2   │ BackupBeforeSave=true
   3   │ ConfigVersion=2
   4   │ DropToBackgroundOnCopy=true
   5   │ HideWindowOnCopy=true
   6   │ MinimizeOnCopy=false
   7   │ MinimizeOnOpenUrl=true
   8   │ UpdateCheckMessageShown=true
```

I have read https://github.com/keepassxreboot/keepassxc/discussions/5879, but I don't think that this is what I want.

**Comment by droidmonkey** [comment, marked as answer]
I think there is a bug on the extension side that was recently introduced and then fixed related to this.

**Reply by stdedos** [reply]
Would you mind linking the ticket? and/or when (version) was it fixed / when is it expected to land?

**Reply by droidmonkey** [reply]
This one https://github.com/keepassxreboot/keepassxc-browser/pull/1999

I also think I don't necessarily understand your problem completely.

**Reply by stdedos** [reply]
I expect that, if KeePassXC is executed, then clicking on a 
![image](https://github.com/keepassxreboot/keepassxc/assets/133706/a69b6e17-b35b-45b3-b2a6-8a3d1afeb845)
icon would trigger unlocking the all of the connected databases.

That does not happen for me after every boot. Even after explicitly clicking the KeePassXC icon on the Application Menu and seeing the main window, that still doesn't happen.
